buildScripts/docker/docker-compose-oracle.yml

```diff
@@ -1,13 +1,14 @@
 version: '3.1'
 
 services:
-  oracle:
-    container_name: oracleDB
-    image: quillbuilduser/oracle-18-xe:latest
-    ports:
-      - "1521"
-    environment:
-      WEB_CONSOLE: "true"
-      DBCA_TOTAL_MEMORY: 1024
-      PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-      USE_UTF8_IF_CHARSET_EMPTY: "true"
\ No newline at end of file
+    oracle:
+        container_name: oracleDB
+        image: gvenzl/oracle-xe:18-slim-faststart
+        ports:
+            - "3003:1521"
+        environment:
+#            WEB_CONSOLE: "true"
+#            DBCA_TOTAL_MEMORY: 1024
+#            PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+#            USE_UTF8_IF_CHARSET_EMPTY: "true"
```

Comment:
Just checking to make sure that these were meant to remain commented out.

Reply:
Yep, let me drop them. They are not required with the new image